src/main/java/steps/TestResultsSteps.java

```diff
@@ -639,6 +654,13 @@ public void postVehicleTestResultsWithAlterations(String requestBody, List<JsonP
         this.response = testResultsClient.postVehicleTestResultsWithAlterations(requestBody, alterations);
     }
 
+    @Step
+    public void postVehicleTestResultsWithNoAuthorization(String requestBody) {
+        setWrongAtuh();
```

Comment:
maybe we should fix the typo in these methods to make it "Auth" instead "Atuh"

Reply:
Changes Done. Please review